webpack/scenes/Subscriptions/SubscriptionConstants.js

```diff
@@ -2,6 +2,14 @@ export const SUBSCRIPTIONS_REQUEST = 'SUBSCRIPTIONS_REQUEST';
 export const SUBSCRIPTIONS_SUCCESS = 'SUBSCRIPTIONS_SUCCESS';
 export const SUBSCRIPTIONS_FAILURE = 'SUBSCRIPTIONS_FAILURE';
 
+export const UPDATE_QUANTITY_REQUEST = 'UPDATE_QUANTITY_REQUEST';
+export const UPDATE_QUANTITY_SUCCESS = 'UPDATE_QUANTITY_SUCCESS';
+export const UPDATE_QUANTITY_FAILURE = 'UPDATE_QUANTITY_FAILURE';
+
+export const SUBSCRIPTIONS_QUANTITIES_REQUEST = 'SUBSCRIPTIONS_QUANTITIES_REQUEST';
+export const SUBSCRIPTIONS_QUANTITIES_SUCCESS = 'SUBSCRIPTIONS_QUANTITIES_SUCCESS';
+export const SUBSCRIPTIONS_QUANTITIES_FAILURE = 'SUBSCRIPTIONS_QUANTITIES_FAILURE';
```

Comment:
I don't think so. The action runs on background so there's no need to block the UI for users. If the start editing prior to successful fetch a spinner is displayed.

Reply:
Sorry, I meant to put this comment under the update actions.  I mean that we need to add ` Actions::Katello::UpstreamSubscriptions::UpdateEntitlement` to the list of `BLOCKING_FOREMAN_TASK_TYPES` below so that the buttons are disabled correctly when this task is in progress.

Reply:
That makes sense. I didn't understand it first. Updated.